CMakeLists.txt

```diff
@@ -149,6 +149,11 @@ if(NOT DAEMON_EXTERNAL_APP)
     else()
         option(USE_CURSES "Enable fancy colors in terminal's output" ON)
     endif()
+
+    if (BUILD_CLIENT OR BUILD_TTY_CLIENT)
```

Comment:
Does the TTY client need mumblelink?

Reply:
I don't know but building TTY currently builds it. Maybe it builds it because we never added any ifdef for it.

But it may make sense that TTY client does mumble link: the same way TTY can be used to do text chat, why not doing voice chat as well… The biggest concern is that the current mumble link is still mostly a stub, whatever the client.